webpack.config.js

```diff
@@ -288,8 +285,7 @@ const commonConfig = {
         splitChunks: {
             name: 'vendor',
             minChunks: 2
-        },
-        noEmitOnErrors: true
```

Comment:
Denne burde vel bruke `emitOnErrors: false` nå i stedet for å fjernes?

Reply:
Mener det skal være default: https://webpack.js.org/configuration/optimization/#optimizationemitonerrors